app/branding.js

```diff
@@ -365,7 +298,7 @@ export function getAssetNamespaces() {
     if (_isTestnet()) {
         return [];
     }
-    return ["OPEN.", "RUDEX.", "GDEX.", "XBTSX.", "CITADEL.", "DEEX."];
+    return ["OPEN.", "RUDEX.", "GDEX.", "XBTSX.", "CITADEL."];
```

Comment:
Remove OPEN?

Reply:
Currently marked as disabled with the ! icon.